src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: $transition-duration ease;
```

Comment:
You should write what parameter will use in this transition
For example: `transition: color 300ms`

Reply:
Hmmm, it worked in the way I wrote so I thought if it made my code shorter I could use it :)
But already checked your notes 👍